### ISSUE TITLE
Report is fully qualified name is used in code

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#2137]
Similar to MiKo_3236 we should report if a fully qualified name is used in code.

Example:
```c#
public string DoSomething() => System.IO.File.ReadAllText("some path");
```

We should report to use only the type and move the qualification to a `using` directive.

```c#
using System.IO;

public string DoSomething() => File.ReadAllText("some path");
```
